server.js

```diff
@@ -1,9 +1,9 @@
 const express = require('express');
+const home = require("./routes/home.js");
+
 const server = express();
 
-server.get('/', (request, response) => {
-   response.send('hello')
-})
+server.get('/', home.get)
```

Comment:
I had to process this a couple of times but I think I see how this works, so...
1) The get function is being exported from the home.js file and then required (imported) to the server.js file
2) The contents of the get function are being "served" by the server.js so is then displayed on the page http://localhost:3000/

Maybe it's a little confusing having the function called "get" cos there are a few gets on the page to follow hahaha
But apart from that, the code is smashing!!!